environments/production/hiera/db.json

```diff
@@ -7,7 +7,7 @@
         "::oaeservice::backup::cassandra"
     ],
 
-    "duplicity::params::excludes": ["**/snapshots"],
+    "duplicity::params::excludes": ["**/snapshots","**/OpsCenter"],
```

Comment:
A reasonable question. I am in fact uncertain if it would match, the docs
are not clear.
Better to make the change I think.


On 4 June 2015 at 12:15, Simon Gaeremynck <notifications@github.com> wrote:

> In environments/production/hiera/db.json
> <https://github.com/oaeproject/puppet-hilary/pull/156#discussion_r31711479>
> :
>
> > @@ -7,7 +7,7 @@
> >          "::oaeservice::backup::cassandra"
> >      ],
> >
> > -    "duplicity::params::excludes": ["**/snapshots"],
> > +    "duplicity::params::excludes": ["**/snapshots","**/OpsCenter"],
>
> Just a dumb question from me:
>
> The backup directory is /data/cassandra/data. The directory we want to
> exclude is /data/cassandra/data/OpsCenter. Will the **/ glob work for
> that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/oaeproject/puppet-hilary/pull/156/files#r31711479>.
>